omod/src/main/webapp/resources/js/aijar.js

```diff
@@ -161,25 +167,33 @@ function validateRequiredField(prime, factor, message_to_throw, input_type) {
     var selected_value = null;
     getField(prime + '.error').html("").hide;
 
-    if (input_type == "select") {
-        selected_value = jq(factor).find(":selected").text().trim().toLowerCase();
-        if (selected_value != '' && getValue(prime + '.value') == '') {
+    if (input_type === "select") {
```

Comment:
@slubwama Can you verify that these changes to not affect the ART Summary page where these functions are used for validation